app/services/sign_in/session_creator.rb

```diff
@@ -55,7 +55,10 @@ def create_new_access_token
         refresh_token_hash:,
         parent_refresh_token_hash:,
         anti_csrf_token:,
-        last_regeneration_time: refresh_created_time
+        last_regeneration_time: refresh_created_time,
+        first_name: user_attributes[:first_name],
```

Comment:
If access token automatically filters the `user_attributes` it needs, we could just do `user_attributes: user_attributes` here